app/views/dashboard/apps/index/appsView.tsx

```diff
@@ -115,21 +219,23 @@ export const AppsView = ({
               )}
             </Tabs.Tab>
 
-            {endpoints && endpoints.length > 0 ? (
-              <Tabs.Tab label="Teams">
+            <Tabs.Tab label="Teams">
+              {notOwnerEndpoints &&
+              notOwnerEndpoints.length > 0 &&
+              userDataByEndpoint.length > 0 ? (
                 <Table
                   columns={["App", "Invite status", "Role", ""]}
-                  data={teams.map((team) => ({
+                  data={(notOwnerEndpoints as ProcessedEndpoint[]).map((team, idx) => ({
                     id: team.id,
                     app: {
-                      value: team.app,
-                      element: <Link to={team.id.toString()}>{team.app}</Link>,
+                      value: team.name,
+                      element: <Link to={team.id.toString()}>{team.name}</Link>,
                     },
                     inviteStatus: {
-                      value: team.accepted ? "Accepted" : "Pending",
-                      element: <StatusTag accepted={team.accepted} />,
+                      value: userDataByEndpoint[idx]?.accepted ? "Accepted" : "Pending",
+                      element: <StatusTag accepted={userDataByEndpoint[idx]!.accepted} />,
```

Comment:
```suggestion
                      element: <StatusTag accepted={userDataByEndpoint[idx]!?.accepted} />,
```
sometimes this happens to be undefined because it's depending on the endpoints array
which is not always in sync with the userDataByEndpoint array
we could try to add a check here